### PR TITLE
[skip-ci] Update readme

### LINKED ISSUE #7311: [DOC] README conda install is for cudf 0.13
As noted in https://stackoverflow.com/questions/66046111/from-numba-import-cuda-numpy-support-and-importerror-cannot-import-name-numpy , we should update the [conda section](https://github.com/rapidsai/cudf#conda) of the README to be the stable release.

### PR DESCRIPTION
**Pull request by shwina**
Closes #7311

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,15 +54,15 @@ Please see the [Demo Docker Repository](https://hub.docker.com/r/rapidsai/rapids
 
 cuDF can be installed with conda ([miniconda](https://conda.io/miniconda.html), or the full [Anaconda distribution](https://www.anaconda.com/download)) from the `rapidsai` channel:
 
-For `cudf version == 0.13` :
+For `cudf version == 0.18` :
```
Comment (kkraus14) [inline README.md:57]: Can we add these to the release script version updater?

Comment (shwina) [reply README.md:57]: Done. Should we just remove this change now?

**Review by kkraus14 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (kkraus14) [reply README.md:57]: Nah, lets leave it.

**Review by ajschmidt8 — APPROVED** [review]

**Comment by kkraus14** [comment]
@gpucibot merge

**Pull request merged**